From 989c31e376b0853b770745f84147dd07ff8c0b23 Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Thu, 23 Sep 2021 15:12:02 +0300
Subject: [PATCH] [SW-35427] gaudi2: image pointer offset change

Changing the SPI flash layout of Gaudi2 legacy boot flow to remove the
holes in between. Changing the image pointer flash offset to match
the new layout as below:

Sector  image
0	BTL primary
1	BTL secondary
2	BTL PRSTN1
3	BTL PRSTN2
4	BTL PCIE1
5	BTL PCIE2
6	Image pointers ARM
7	UENV (reservd for any internal testing procedure)
8	Preboot1
9	Prebbot2

Change-Id: I896aa55b225a0338e59a241b151e9feb991665b6
---

diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index da1e1f9..02be606 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -25,7 +25,7 @@
 #define SDRAM_BASE			0x7FFD000000ull
 #endif
 
-#define HL_FW_BOOT_PTRS_BASE		0x80000
+#define HL_FW_BOOT_PTRS_BASE		0x60000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
